docs/technical-details/release-notes/4.tucker/README.md

```diff
@@ -17,9 +19,13 @@ HarperDB version 4 ([Tucker release](./tucker.md)) represents major step forward
 * Native caching capabilities for high-performance cache scenarios
 * Clone node functionality
 
-## [4.3](./4.3.0.md)
-* Relationships, joins, and broad new querying capabilities for complex and nested conditions, sorting, joining, and selecting with significant query optimizations
-* More advanced transaction support for CRDTs and storage of large integers (with BigInt)
-* Better management with new upgraded local studio and new CLI features
+## [4.1](./4.1.0.md)
+* New streaming iterators mechanism that allows query results to be delivered to clients _while_ querying results are being processed, for incredibly fast time-to-first-byte and concurrent processing/delivery
```

Comment:
@jcohen-hdb @kriszyp 
 
 A new mechanism that streams query results to clients as they're being processed, providing ultra-fast time-to-first-byte and allowing for concurrent processing and delivery of data.
 
 An updated concurrency model that leverages threads for more efficient resource usage, improving overall performance and scalability.